code/modules/mob/living/simple_animal/hostile/human/vox.dm

```diff
@@ -16,6 +21,298 @@
 
 	corpse = /obj/effect/landmark/corpse/vox
 
+/mob/living/simple_animal/hostile/humanoid/vox/New() // vox hostiles will speak chikun
+	..()
+	languages += all_languages[LANGUAGE_VOX]
+
+///////////////////////////////////////////////////////////////////NU VOX RAIDERS///////////
+//Armed with various ranged weapons, with a bias towards cheaper ballistics
+
+/mob/living/simple_animal/hostile/humanoid/vox/spaceraider //Baseline chikun raider, here so that the others that follow inherit all its useful properties
```

Comment:
disappointed that you can't distract them or turn them neutral by throwing credits at them

Reply:
I'm not sure how to code this, but I'm willing to try. Even if it only works for a half second it could be a fun detail.